### PR TITLE
Report: header sticks to top

### PR DESCRIPTION
**Pull request by ebidel**
R: all

This sticks the report header to the top of the page as you scroll. I think this is a nice change to make the share/print button always visible (e.g. LH viewer, devtools).

### CODE REVIEW

**Comment by ebidel** [comment]
Don't look at this yet. Not ready.

**Comment by ebidel** [comment]
Ready

File: lighthouse-core/report/styles/report.css [hunk 1]
```diff
@@ -78,6 +82,11 @@ html, body {
   height: 100%;
 }
 
+/* When deep linking to a section, bump the headind down so it's not covered by the header. */
```
Comment (paulirish) [inline lighthouse-core/report/styles/report.css]: headind typo

File: lighthouse-core/report/styles/report.css [hunk 2]
```diff
@@ -181,9 +194,9 @@ body {
 }
 
 .menu {
-  width: 22%;
+  width: var(--report-menu-width);
   background: #FFFFFF;
-  min-width: 185px;
+  /*min-width: 185px;*/
```
Comment (paulirish) [inline lighthouse-core/report/styles/report.css]: ^ ^

File: lighthouse-core/report/styles/report.css [hunk 3]
```diff
@@ -307,15 +320,19 @@ body {
...
   background: #FAFAFA;
-  margin-left: 22%;
+  margin-left: var(--report-menu-width);
   display: flex;
   flex-direction: row;
   align-items: center;
   justify-content: flex-end;
   padding: 0 var(--heading-line-height);
+  position: fixed;
+  z-index: 1;
```
Comment (paulirish) [inline lighthouse-core/report/styles/report.css]: do me a solid and add `will-change: transform` on here as well? for lo-dpi chrome desktop users.

File: lighthouse-core/report/styles/report.css [hunk 4]
```diff
@@ -164,15 +173,19 @@ body {
...
+  padding-top: var(--report-header-height);
+}
+
 .report-body__menu-container {
   height: 100%;
   width: 100%;
   min-width: 230px;
-  max-width: 1280px;
+  max-width: var(--report-width);
   position: fixed;
```
Comment (paulirish) [inline lighthouse-core/report/styles/report.css]: can u add will-change to this one as well?

File: lighthouse-core/report/styles/report.css (hunk 1 above)
Comment (ebidel) [inline lighthouse-core/report/styles/report.css]: done

File: lighthouse-core/report/styles/report.css (hunk 2 above)
Comment (ebidel) [inline lighthouse-core/report/styles/report.css]: done

File: lighthouse-core/report/styles/report.css (hunk 3 above)
Comment (ebidel) [inline lighthouse-core/report/styles/report.css]: done

**Comment by ebidel** [comment]
PTAL

**Pull request merged**